app/controllers/groups_controller.rb

```diff
@@ -294,6 +327,23 @@ def add_members
     end
   end
 
+  def membership_request
```

Comment:
hmm the method name here  doesn't seem to reflect what the method actually does:
1. Adds a user to the group 
2. Delete all the user's existing requests to a group.